docs/documentation/development/notes.md

```diff
@@ -20,4 +20,14 @@ The manifest generation could be triggered by:
 The "nice manifest" option will add line breaks after every imported package.
 Now you can have a look at the manifest that would be generated by the bndtool using:
 
-    cat target/classes/META-INF/MANIFEST.MF
\ No newline at end of file
+    cat target/classes/META-INF/MANIFEST.MF
+
+Currently you need to run the maven build every time code is changed in the files. That can be quite cumbersome during development. The easiest way to tackle that is to keep a separate copy of index.html that references all js/html files (and respective libraries) inside web-src folder. You will have to change:
```

Comment:
`a separate copy of index.html` -> can we commit this file under a different name (and exclude it from the packaging), e.g. `index_dev.html`? Or wouldn't this work because the name has to be `index.html` in order to work correctly?